target-platform/target-platform.target

```diff
@@ -37,7 +37,7 @@
 	<unit id="org.eclipse.mylyn.wikitext_feature.feature.group" version="0.0.0"/>
 </location>
 <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-	<repository location="https://download.eclipse.org/lsp4e/releases/latest/"/>
+	<repository location="https://download.eclipse.org/lsp4e/snapshots/"/>
```

Comment:
Wouldn't it be good to put a FIXME, to change to latest, after a new release of lsp4e?

Reply:
I don't think it changes anything; I'm usually the one doing releases and I don't look for "FIXME" anyway.

Reply:
If you're willing to contribute that, it would be highly welcome.